peachjam/adapters/adapters.py

```diff
@@ -225,7 +230,9 @@ def update_document(self, url):
             "date": datetime.strptime(document["expression_date"], "%Y-%m-%d").date(),
         }
         if document["locality"]:
-            frbr_uri_data["locality"] = Locality.objects.get(code=document["locality"])
+            frbr_uri_data["locality"] = Locality.objects.gapi_urlet(
```

Comment:
??

Reply:
yikes :grimacing: i need to get used to using git commit -p